### ISSUE TITLE
OpenApi UI not displaying request body examples

### ISSUE THREAD

**Issue by cavalka** [OpenLiberty/open-liberty#6637]
The OpenApi UI is not displaying example objects for request body.  Below is generated Open API document.  It displays examples in scheme BasicClientRegistration.  The examples in requestBody should override the scheme examples. 

```
paths:
  /API/CR03:
    put:
      tags:
      - CR03
      summary: Update basic client registration details
      description: This will execute CR03 update transaction
      operationId: updateCR03
      requestBody:
        description: Updated basic client registration object
        content:
          application/json:
            schema:
              $ref: '#/components/schemas/BasicClientRegistration'
            examples:
              Client:
                summary: Client update example
                value: <html><body><ul><li>item 1</li><li>item 2</li></ul></body></html>
          application/xml:
            schema:
              $ref: '#/components/schemas/BasicClientRegistration'
        required: true
      responses:
        200:
          description: Client updated.
          content:
            application/json:
              schema:
                $ref: '#/components/schemas/BasicClientRegistration'
              examples:
                Client:
                  summary: Client update example
                  value: <html><body><ul><li>item 1</li><li>item 2</li></ul></body></html>
            application/xml:
              schema:
                $ref: '#/components/schemas/BasicClientRegistration'

```

**Comment by yeekangc** [comment]
@arthurdm @arturdzm

**Comment by arturdzm** [comment]
This seems to be case of missing functionality in the UI and would require new UI components developed and added.
At the moment there is no way to render and select examples property, as there could be multiple examples for each media type. 

However, the **example** field seems to work as designed.

Here is relevant issue  Swagger https://github.com/swagger-api/swagger-ui/issues/2651

**Comment by Eng-Fouad** [comment]
It is now implemented in Swagger UI 3.23.0:

swagger-api/swagger-ui#2651

**Comment by Azquelt** [comment]
This now works since the UI was updated in #9880
